pandas/core/indexes/accessors.py

```diff
@@ -245,15 +210,34 @@ class PeriodProperties(Properties):
 
 
 class CombinedDatetimelikeProperties(DatetimeProperties, TimedeltaProperties):
-    # This class is never instantiated, and exists solely for the benefit of
-    # the Series.dt class property. For Series objects, .dt will always be one
-    # of the more specific classes above.
-    __doc__ = DatetimeProperties.__doc__
 
-    @classmethod
-    def _make_accessor(cls, data):
+    def __new__(cls, data):
```

Comment:
can we be consistent for validation, IOW either do it in ``__new__`` or in ``__init__``?

Reply:
I had to put this in `__new__` since we don't instantiate a `CombinedDatetimelikeProperties`, just one of its parents. I *think* this is unavoidable, but it would be nice to make it more consistent with the others. I'll see what I can do.

Reply:
One option is to write a `Properties._validate` do most of the work here. But it feels weird to have a base class decide which child class should be instantiated. I think having it in `__new__` (with a comment for why it's there) is clearest.